source/release-notes/3.6-upgrade-sharded-cluster.txt

```diff
@@ -40,6 +40,14 @@ clients to connect to your sharded cluster.
 
 For more information, see :ref:`3.6-bind_ip-compatibility`
 
+Shard and Config Servers Require Replication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in MongoDB 3.6, :program:`mongod` instances with the shard or
+config server role must be replica set members. The following procedure
```

Comment:
I believe starting in 3.4, config servers  had to be replica sets

https://docs.mongodb.com/manual/release-notes/3.4/#removal-of-support-for-sccc-config-servers

Reply:
Removed with the assumption 3.4 config servers will already be running with replication.  Alternatively, I could add a separate paragraph covering the 3.4 config server behavior.